vms/vms/views.py

```diff
@@ -0,0 +1,24 @@
+import requests
+import json
+from shift.utils import create_event_with_details
+from django.http import HttpResponseRedirect
+from django.core.urlresolvers import reverse
+from datetime import datetime
+
+def get_meetup(request):
+    date = str(datetime.today.date())
+    data = {'date': date}
+    response = requests.post('http://127.0.0.1:8080/meetup/api/v1/request_meetup_data/', data)
```

Comment:
the server IP should be dynamic IMO might change with deployment and test server too.

Reply:
Currently, both are not deployed so that depends upon the port which user enters, this will be changed while deployment.

Reply:
this method is used in test too right? during test this url will change every time we run the tests its not localhost. We need to keep something dynamic here to cater the prod, dev and test at the same time.

Reply:
https://docs.djangoproject.com/en/dev/ref/contrib/sites/#hooking-into-the-current-site-from-views

Take a look at this once.